src/adv.py

```diff
@@ -1,5 +1,6 @@
+import time
```

Comment:
Make more commits! I know you are new to this, break every functional code into individual commits. Showcasing how you built your code.